trueos-handbook/introducing.rst

```diff
@@ -1095,15 +1213,14 @@ Laptops
 -------
 
 Many |trueos| users successfully run |trueos| on their laptops. However,
-some issues may occur, depending upon the model of laptop. These
-typically deal with:
+some issues may occur, depending upon the model of laptop. Some typical
+laptop issues;
```

Comment:
; -> :

Reply:
I thought this was some sort of odd smiley when I first saw it. Changed the ';' to a ':'.